R/setThreadCount.r

```diff
@@ -23,14 +25,17 @@
 #' setThreadCount(benchmarkData, "nafill", "recommended")
 #' }
 
-setThreadCount <- function(benchmarkData, functionName, type = "recommended")
+setThreadCount <- function(benchmarkData, functionName, type = "recommended", verbose = FALSE)
 {
   setDTthreads(
     if(type == "optimal")
     {
       fastestMedianTime <- benchmarkData[expr == functionName, .(median = min(median))]
       bestThreadCount <- benchmarkData[expr == functionName & median == fastestMedianTime$median, threadCount]
-      cat(sprintf("The number of threads that data.table will use has been set to %d, the thread count that achieved the best runtime for data.table::%s() based on the performed benchmarks.\n", bestThreadCount, functionName))
+      if(verbose) 
+      {
+        message("The number of threads that data.table will use has been set to ", bestThreadCount, ", the thread count that achieved the best runtime for data.table::", functionName, "() based on the performed benchmarks.")
```

Comment:
for translation of long strings it is better to use message(sprintf("long %d string with %s", var1, var2)) (one string to translate) instead of message("long", var1, "string with", var2) (two strings to translate, more difficult and maybe impossible because each is without context of the other)